workflow/apm/database.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Databases
+sidebarTitle: Databases
+---
+
+The Databases page displays all of your database data that flows through your instrumented APMs. This page is useful for monitoring high-level database interactions and how they can directly impact the performance of other key applications within your software architecture. 
+
+Each database operation is comprised of tags and queries. Tags and queries are useful for drilling into specific database operations across your microservice architecture as described [here](). Easily track the total number of database requests over a specific period, the average query time for individual queries, and the total number of queries (throughput) your system can handle per second.
```

Comment:
Not everything is a microservice architecture, so we can't say that here. I would recommend rephrasing to "...database operations across your instrumented applications."